chapters/Gosses1.md

```diff
@@ -0,0 +1,39 @@
+# Standing on the Shoulders of the Guy in the UK Office
+ 
+Justin Gosses
+
+> Writing in progress....  &mdash; 
+
+
+The phrase “If I have seen further, it is by standing on the shoulders of giants.” was written by Isaac Newton to Robert Hooke in reference to building on the work his predecessors. In science, we often think of these giants as long dead men in history books. In computer science, many of the giants are still alive.  In open-source, they are middle aged to young people you end up drinking beer with at conferences. In intra-source, they are your co-workers writing code right now, including those you’ve never met. 
+ 
+If there is a low hanging fruit of innovation, it might be intra-source. While open-source has proved its value in terms of better code through more eyeballs and the ability to leverage mindshare to organize development of code ecosystems without centralized control, open-source is hard for companies, and especially hard for oil and gas. Lawyers get involved. Release processes are slow, non-existent, or written for the worst-case scenario. Intra-source offers many of the benefits with increased speed and less institutional pushback. 
+ 
+Intra-source refers to the sharing and reuse of code written in an organization by any other part of the organization. While oil and gas companies are adept at sharing within disciplines geographically distributed in multiple offices, up and down the chain of command hand-off style, or across disciplines operating in a local area, intra-source needs to be code sharing across the entire company. Financial analysts and geoscientists both use Pandas, a data analysis library. Petrophysicists and procurements both had an interest in natural language processing. Engineers building monitoring software for offshore installments and in-store marketing analysts might be both using OpenCV for image recognition. Traditional boundaries in oil and gas aren't helpful when it comes to sharing code. The distinguishing characteristics are tools, not disciplines or position with the exploration to marketing stream. 
```

Comment:
I have seen this more common as `inner source`ing:
https://en.wikipedia.org/wiki/Inner_source